tests/drivers/flash/common/testcase.yaml

```diff
@@ -31,13 +31,15 @@ tests:
       - nrf52840dk/nrf52840
   drivers.flash.common.default:
     filter: ((CONFIG_FLASH_HAS_DRIVER_ENABLED and not CONFIG_TRUSTED_EXECUTION_NONSECURE)
-      and dt_label_with_parent_compat_enabled("storage_partition", "fixed-partitions"))
+      and (dt_label_with_parent_compat_enabled("storage_partition", "fixed-partitions")
+      or dt_label_with_parent_compat_enabled("storage_partition", "nordic,owned-partitions")))
```

Comment:
@PerMac Is it expected that
`dt_label_with_parent_compat_enabled("storage_partition", "fixed-partitions")`
returns False for node that has
`compatible = "nordic,owned-partitions", "fixed-partitions";`
???

Reply:
I opened #75559 in an attempt to address this.